docker-compose.yml

```diff
@@ -27,7 +27,7 @@ services:
         volumes:
             - 'postgres_data:/postgres/data'
         ports:
-            - 5432
+            - 5432:5432
```

Comment:
Not strictly necessary, but useful if you want to run the bot locally

Reply:
Interesting... according to https://docs.docker.com/compose/compose-file/compose-file-v3/#ports, without `:5432` Docker picks a random port, which would work around your port in use issue. I suspect the port was actually in use.